.github/ISSUE_TEMPLATE/epic.yml

```diff
@@ -0,0 +1,104 @@
+    placeholder: Defines clearly when the work is complete. Acts as a litmus test for "done" and avoids "done" being ambiguous. Useful for implicit assumptions, e.g. ensuring docs updates are not forgotten.
+  validations:
+    required: true
+- type: textarea
+  id: measurement
+  attributes:
+    label: Measurement
+    description: How will we know whether we've been successful / solved the problem? How will you measure the success of the epic? Ideally this metric is one of our key product metrics.
+    placeholder: Important as it's how we track the outcomes (not just output) of the work and prove a change was worth it. Or it should be removed or iterated.
+  validations:
+    required: true
+- type: textarea
+  id: growth-area
+  attributes:
+    label: Growth Area
+    description: Which aspect of Gitpod do we expect improvements in? Acquisition/Onboarding/Exploration/Expansion as defined in [Funnel Proposal](https://www.notion.so/gitpod/Funnel-Proposal-d7d0dba8aced4184b660092a74f8dd3a) (internal)
+    placeholder: Growth is key. This allows us to frame epics from a growth context. Which areas are we expecting this epic to help us with our growth initiatives?
+  validations:
+    required: false
+- type: textarea
+  id: personas
+  attributes:
+    label: Persona(s)
+    description: Who will be impacted by this change? Which of our personas will be impacted by this change?
+    placeholder: Why? To bring persona's into our work. Persona's can help us prioritise our markets. Currently, we are not focusing on the education/training persona currently. We should avoid epics which target this persona.
+  validations:
+    required: false
+- type: textarea
+  id: hypothesis
+  attributes:
+    label: Hypothesis
+    description: If we do X, we expect Y
+    placeholder: Can be useful if the work is explicitly experimental.
+  validations:
+    required: false
+- type: textarea
+  id: in-scope
+  attributes:
+    label: In scope
+    description: Explicitly define the items in scope.
+    placeholder: Optional, sometimes is useful for explicitness.
+  validations:
+    required: false
+- type: textarea
+  id: out-of-scope
+  attributes:
+    label: Out of scope
+    description: Explicitly define the items out of scope.
+    placeholder: Optional, sometimes is useful for explicitness.
+  validations:
+    required: false
+- type: textarea
+  id: complexities
+  attributes:
+    label: Complexities
+    description: Discuss any known complexities
+    placeholder: Optional, sometimes is useful for explicitness.
+  validations:
+    required: false
+- type: textarea
+  id: press-release
+  attributes:
+    label: Press release
+    description: Create excitement about the idea
+    placeholder: Useful if you want to spend the extra time to get stakeholders, the team, or customers excited.
+  validations:
+    required: false
```

Comment:
I'd suggest we remove this, I think the original intent for PRFAQ process was misunderstood, I don't think this makes sense to do as part of our epic template. 

```suggestion
```